src/__tests__/Column.test.tsx

```diff
@@ -1,41 +1,23 @@
-import React
-  from 'react';
-import Column, {enumDrawType} from '../components/Column';
+import React, {ReactDOM} from 'react';
+import {shallow} from 'enzyme';
+import Column from '../components/Column';
 
 describe('<Column /> Unit Tests', () => {
-  let columnString: Column;
-  let columnNum: Column;
-  /**
-   * Creates a column object for test cases
-   */
-  beforeEach(() => {
-    // eslint-disable-next-line max-len
-    columnString = new Column('string', enumDrawType.occurrence, 'heart attacks');
-    columnNum = new Column('number', enumDrawType.any, 'sandwiches');
-  });
-
   describe('constructor()', () => {
-    it('Constructor assigns variables', () => {
-      // tests for string data
-      expect(columnString.primType).toBe('string');
```

Comment:
why are these unit tests removed?

Reply:
probably because Amanda's branch is out of date.

Reply:
just copied file from develop branch to this branch, should be ok